libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -26,6 +26,9 @@ extern const AP_HAL::HAL &hal;
 // parameter defaults
 const float OA_LOOKAHEAD_DEFAULT = 15;
 const float OA_MARGIN_MAX_DEFAULT = 5;
+#if APM_BUILD_TYPE(APM_BUILD_Rover)
```

Comment:
we don't really need this if here but it's OK.. just for next time ..

Reply:
@rmackay9 I actually had to force push this one as a specific change later because Travis CI was failing , stating "unused variable"